lib/release-ops.js

```diff
@@ -202,19 +204,17 @@ function calculateReleaseFromGitLogs(currentVersion, logs, prereleaseId) {
      * @returns {string} A line for a changelog with a link to the GitHub commit
      */
     function generateChangelogLine(log) {
-        return log.raw.replace(/^\* ([0-9a-f]{40})/, function(_, hash) {
-            return "* [`" + hash.slice(0, 7) + "`](https://github.com/" + repository + "/commit/" + hash + ")";
-        });
+        return log.raw.replace(/^\* ([0-9a-f]{40})/, (_, hash) => `* [\`${hash.slice(0, 7)}\`](https://github.com/${repository}/commit/${hash})`);
```

Comment:
I think this would be a little more readable:

```suggestion
        return log.raw.replace(
            /^\* ([0-9a-f]{40})/,
            (_, hash) => `* [\`${hash.slice(0, 7)}\`](https://github.com/${repository}/commit/${hash})`
        );
```